guides/_subsections/configure-jmx-credentials-storage.md

```diff
@@ -0,0 +1,44 @@
+## [Configure JMX Credentials Storage](#configure-jmx-credentials-storage)
+
+Target JVM applications may require Cryostat to pass an authentication challenge before being able to communicate over JMX
+and manage JFR. Cryostat has two supported mechanisms for these JMX credentials:
+<ol>
+  <li>
+    <b>JMX Credentials Keyring</b>: see <a href="#store-jmx-credentials">Store JMX Credentials</a> for more detail.
+    This mechanism entails uploading a JMX Credentials definition to the Cryostat backend keyring storage. Cryostat
+    automatically checks the keyring for credentials matching a target application when a request to that application
+    is opened. If no credentials are found, Cryostat responds to the requesting client (ex. the Cryostat Web UI) with
+    a response indicating the JMX authentication failure.
+  </li>
+  <li>
+    <b>X-JMX-Authorization</b> header passthrough: when the Cryostat Web UI receives a JMX authentication failure response
+    from the Cryostat server it displays a prompt asking for JMX credentials. Depending on the configuration set in this
```

Comment:
`from the Cryostat server it displays` -> `from the Cryostat server, it displays`